src/components/popover/widgets/capture-keystroke/capture-keystroke-button.component.html

```diff
@@ -1,6 +1,7 @@
 <button type="button" class="btn btn-sm btn--capture-keystroke"
             [ngClass]="{'btn-default': !record, 'btn-info': record}"
-            (click)="start()"
+            (keydown.enter)="onButtonEnterDown($event)"
```

Comment:
With `Sometimes easier is not necessary better.` I was referring to your statement `this was easiest solution for me to fix those bugs`. 

I would suggest that we go trough this code on a hangout. One capture does not emit two events.

I will create PR in the next days for this popover esc problem. I think that service is ok, I created PR #231 with same service and everything work correctly.